zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterEventPoller.java

```diff
@@ -214,6 +222,12 @@ public void run() {
     }
   }
 
+  private String getInterpreterSettingId() {
+    String id = interpreterGroup.getId();
+    int indexOfColon = id.indexOf(":");
+    return id.substring(0, indexOfColon);
+  }
```

Comment:
I think it is better to put `getInterpreterSettingId` into `Utils` as well because it is about how we parse interpreterGroup